serve.go

```diff
@@ -501,6 +506,15 @@ type config struct {
 	err error
 }
 
+func (c *config) setupCommandCallback() {
```

Comment:
Also: I hear you about the multiple consumers, I get it. We could cut the pubsub out of it altogether and just callback directly from the queue package, but I do like the separation of duties when using the pubsub queue.